src/App.jsx

```diff
@@ -6,23 +6,32 @@ import Nav from './Components/Nav'
 import Topics from './Components/Topics'
```

Comment:
Currently your input field for comment posting appears at an other place where it's not supposed to:
<img width="795" alt="Screenshot 2023-06-01 at 16 15 14" src="https://github.com/Tomoneill98/fe-nc-news/assets/54496822/b2465df6-0f14-4990-bdaa-242785eecc58">